queens-psyc-clinic/app
# [FE FIX] change location field from drop-down to text

### **FE FIX - changing location from a dropdown to input field**

**Testing**
1. navigate to branch: `git checkout location-fix`
2. `cd` to the `public` folder
3. run `npm install`
4. run `npm start`
5. ensure on the admin dashboard page when you press the add + button on the top right it opens the add modal and the location is now an input field not a dropdown.
<br>

For reference checkout issue: #104
